marketplaceItems/appreciate/appResources/appData/appreciate_app.js

```diff
@@ -0,0 +1,947 @@
+                currentInitialAngularVelocity.z = Math.random() *
+                    (INTENSITY_ENTITY_MAX_ANGULAR_VELOCITY.z -
+                    INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.z) + INTENSITY_ENTITY_MIN_ANGULAR_VELOCITY.z;
+                props.angularVelocity = currentInitialAngularVelocity;
+
+                intensityEntity = Entities.addEntity(props, "avatar");
+                
+                var materialProps = INTENSITY_MATERIAL_ENTITY_PROPS;
+                materialProps.parentID = intensityEntity;
+                intensityMaterialEntity = Entities.addEntity(materialProps, "avatar");
+            }
+        } else {
+            if (intensityEntity) {
+                Entities.deleteEntity(intensityEntity);
+                intensityEntity = false;
+            }
+            if (intensityMaterialEntity) {
+                Entities.deleteEntity(intensityMaterialEntity);
+                intensityMaterialEntity = false;
+            }
+            
+            maybeClearUpdateIntensityEntityInterval();
+        }
+    }
+
+    // Function that AppUI calls when the App's UI opens
+    function onOpened() {
+        updateCurrentIntensityUI();
+    }
+
+    // Locally pre-caches all of the sounds in the sounds/claps and sounds/whistles
+    // directories.
+    var NUM_CLAP_SOUNDS = 16;
+    var NUM_WHISTLE_SOUNDS = 17;
+    var clapSounds = [];
+    var whistleSounds = [];
+    function getSounds() {
+        for (var i = 1; i < NUM_CLAP_SOUNDS + 1; i++) {
+            clapSounds.push(SoundCache.getSound(Script.resolvePath(
+                "resources/sounds/claps/" + ("0" + i).slice(-2) + ".wav")));
+        }
+        for (i = 1; i < NUM_WHISTLE_SOUNDS + 1; i++) {
+            whistleSounds.push(SoundCache.getSound(Script.resolvePath(
+                "resources/sounds/whistles/" + ("0" + i).slice(-2) + ".wav")));
+        }
+    }
+
+    // Locally pre-caches the Cheering and Clapping animations
+    var whistlingAnimation = false;
+    var clappingAnimation = false;
+    function getAnimations() {
+        var animationURL = Script.resolvePath("resources/animations/Cheering.fbx");
+        var resource = AnimationCache.prefetch(animationURL);
+        var animation = AnimationCache.getAnimation(animationURL);
+        whistlingAnimation = { url: animationURL, animation: animation, resource: resource};
+
+        animationURL = Script.resolvePath("resources/animations/Clapping.fbx");
+        resource = AnimationCache.prefetch(animationURL);
+        animation = AnimationCache.getAnimation(animationURL);
+        clappingAnimation = { url: animationURL, animation: animation, resource: resource};
+    }
+
+    // If we're currently fading out the appreciation sounds on an interval,
+    // clear that interval.
+    function maybeClearSoundFadeInterval() {
+        if (soundFadeInterval) {
+            Script.clearInterval(soundFadeInterval);
+            soundFadeInterval = false;
+        }
+    }
+
```

Comment:
Can you check the file for two spaces after functions?

Reply:
not seeing anything on my end

Reply:
sorry when I said 2 spaces, i meant 2 lines between functions per the coding standard

Reply:
ahhh ok yes ty